joint:
  names: [pan, tilt]
motion:
  names: [right, zero, left]
  right:
    step: [-3.14, -3.14]  # radian
    time_from_start: 2.0  # sec
  zero:
    step: [0.0, 0.0]
    time_from_start: 3.0
  left:
    step: [3.14, 3.14]
    time_from_start: 6.0

motor_spec:
    gear_ratio : 15
    wheel_base : 0.591
    wheel_radius : 0.11
    enc_pulse : 4000

use_imu_during_odom_calc:
    use_imu : False
    complementary_filter_coef : 0.5